blocks/key-facts/key-facts.js

```diff
@@ -6,40 +9,95 @@ function stripEmptyTags(main, child) {
   }
 }
 
-export default function decorate(block) {
+const blockName = 'key-facts';
+const variantClasses = ['trailing-line'];
+
+const keyFactsColumns = (el, v2) => {
+  el.classList.add(`${blockName}__key-item`);
+
+  // adding icon
+  const icon = el.querySelector('i, .icon');
+
+  if (icon) {
+    el.prepend(icon);
+    icon.classList.add(`${blockName}__icon`);
+  }
+
+  // clearing empty paragraphs
+  const paragraphs = el.querySelectorAll('p');
+  paragraphs.forEach((paragraph) => {
+    stripEmptyTags(el, paragraph);
+  });
+
+  const preSubheading = el.querySelector(':scope > :not(strong, i):first-child');
+  preSubheading?.classList.add('sub-2');
+
+  // find and split number/unit
+  const value = el.querySelector('strong:only-child');
+  // separate number and unit. e.g. up | +31% | Freight efficiency, or 120,000 | psi,
+  const [, prenumber, number, postnumber, text] = value.innerHTML.match('([\\+,\\-]*)([0-9,\\.,]+)([\\%]*) *(.*)');
+  let newValue = null;
+
+  if (value) {
+    if (v2) {
+      newValue = `<span class="${blockName}__number">${prenumber}${number}${postnumber} ${text}</span>`;
+    } else {
+      newValue = `
+        <span class="${blockName}__number">
+          <span class="${blockName}__number--small">${prenumber}</span>
+          ${number}
+          <span class="${blockName}__number--small">${postnumber}</span>
+        </span>
+      `;
+    }
+
+    const numberFragment = document.createRange().createContextualFragment(newValue);
+    value.innerHTML = '';
+    value.append(...numberFragment.childNodes);
+    value.classList.add(`${blockName}__number-wrapper`, 'h2');
+
+    if (!v2 && text) {
+      const unit = createElement('strong', [`${blockName}__unit`, 'sub-2']);
```

Comment:
`createElement ` is now changed to accept classes as options, can you change this across file

Reply:
Good catch :) 
fixed